.github/workflows/client-ci.yml

```diff
@@ -0,0 +1,19 @@
+name: client-ci 
+on:
+  push:
+    branches:
+      - main
+jobs:
+  clientdocs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: [ '3.7', '3.8', '3.9', '3.10' ]
```

Comment:
Choose one. The other branch uses multiple because it tests against them all. We're just building one right?

Reply:
which one should we build with?

Reply:
can do 3.10 if it works